### DISCUSSION TITLE
不知能否添加Nvidia新出的sdr转hdr

### DISCUSSION THREAD

**Discussion by JoeZhangYN** [hooke007/mpv_PlayKit#414] [mpv 通用]
虽然尝试了一下，感觉和原生的hdr视频没办法比，但好歹似乎好像有那么一点提升。不知能否将此功能添加进懒人包里。

**Comment by hooke007** [comment]
驱动/API级的功能请求应向正确的仓库发出。

https://github.com/mpv-player/mpv/issues/13352
